chore(trace): Don't call old trace endpoint from event details

wmak opened the pull request:
- As far as I can tell we don't use quickTrace at all in event details, and its 2 fairly expensive queries, removing them entirely

wmak committed:
chore(trace): Remove quickTrace from eventDetails

- As far as I can tell we don't use quickTrace at all in event details,
  and its 2 fairly expensive queries, removing them entirely

scttcper reviewed (changes requested):
DM'd you about the ANR component

wmak committed:
Merge branch 'master' into wmak/chore/remove-quicktrace-from-eventdetails

wmak committed:
fix: Move quick trace query to only happen for ANR

- Don't hit the lite response anymore either cause we don't need it
- Add an option to pass spans at the top level

wmak committed:
fix: typing

wmak committed:
fix: tests

wmak committed:
fix: make skipLight optional for typing

scttcper reviewed (approved):
🙏
